pytest_fixtures/satellite_auth.py

```diff
@@ -84,6 +95,27 @@ def auth_source_ipa(module_org, module_loc, ipa_data):
     ).create()
 
 
+@fixture(scope='function')
+def auth_source_open_ldap(module_org, module_loc, open_ldap_data):
+    return entities.AuthSourceLDAP(
+        onthefly_register=True,
+        account=open_ldap_data['ldap_user_name'],
+        account_password=open_ldap_data['ldap_user_passwd'],
+        base_dn=open_ldap_data['base_dn'],
+        attr_firstname=LDAP_ATTR['firstname'],
+        attr_lastname=LDAP_ATTR['surname'],
+        attr_login=LDAP_ATTR['login'],
+        server_type=LDAP_SERVER_TYPE['API']['posix'],
+        attr_mail=LDAP_ATTR['mail'],
+        name=gen_string('alpha'),
+        host=open_ldap_data['ldap_hostname'],
+        tls=False,
+        port='389',
```

Comment:
These are dynamic properties, I really not think we should keep them in settings, as a base on our test we can change them as well. What are the benefits we can see if we put them in settings? We can end-up with multiple combinations of settings. 

Not sure can we parameterize the fixture? if yes that is a good thing with observation. In case if we want https auth source we can pass that from the test.